torch/_functorch/make_functional.py

```diff
@@ -4,12 +4,25 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import copy
+from collections import OrderedDict
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    NoReturn,
+    Sequence,
+    Tuple,
+    Type,
+    Union,
+)
+
```

Comment:
Is it possible to add this file to lintrunner.toml so we can check in CI that the type annotations are correct? https://github.com/pytorch/pytorch/blob/master/.lintrunner.toml

Reply:
Done.